.rustfmt.toml

```diff
@@ -0,0 +1,12 @@
+# Disable reordering package imports alphabetically
+reorder_imports=false
+
+# Disable the granular width settings automatically configured following max_width.
+use_small_heuristics="Off"
+
+# Maximum width of each line.
+max_width=200
+
+# Maximum width of each function call
+# It is needed as use_small_heuristics=Off.
+fn_call_width=80
```

Comment:
Is the reformat really worth it or can we rather go with the defaults to prevent git changes for all files?

Reply:
`cargo fmt` shows no changes for me, do have a custom global config that causes the changes?